raspyrfm_client/device/manufacturer/hama/_121938.py

```diff
@@ -0,0 +1,47 @@
+from raspyrfm_client.device import actions
+from raspyrfm_client.device.base import Device
+
+
+class HAMA(Device):
```

Comment:
Please have a look to this commit:

https://github.com/Phunkafizer/raspyrfm-client/commit/66a671c1b74979a1defa87a3e0199998de290b8e

For me looks now much easier, what do you think?